.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -4,8 +4,8 @@ contact_links:
     url: https://github.com/iree-org/iree/issues/new
     about: If you know what you're doing (especially collaborators)
   - name: 📧 Send an email to our mailing list
-    url: https://groups.google.com/forum/#!forum/iree-discuss
-    about: For announcements and asynchronous discussion
+    url: https://lists.lfaidata.foundation/g/iree-technical-discussion
```

Comment:
🤔 I know you've been trying to get these lists renamed to have different letter casing, but this is still redirecting to https://lists.lfaidata.foundation/g/IREE-Technical-Discussion . Seems fine I guess 😛

Reply:
Yeah... At least no capital letters in that particular mailing lists...